src/wallet/components/WalletBottomSheet.tsx

```diff
@@ -0,0 +1,73 @@
+import {
+  Children,
+  cloneElement,
+  isValidElement,
+  useCallback,
+  useMemo,
+} from 'react';
+import { useAccount } from 'wagmi';
+import { Identity } from '../../identity/components/Identity';
+import { background, cn } from '../../styles/theme';
+import type { WalletBottomSheetReact } from '../types';
+import { useWalletContext } from './WalletProvider';
+
+export function WalletBottomSheet({
```

Comment:
is BottomSheet a common term for those kind of components? just a curiosity.

Reply:
i feel like i've seen `Drawer` or `Tray` used more frequently but `WalletBottomSheet` was in the designs

Reply:
Mmmm, it's not a public API so it's fine for now, but in general feel free to triple check those names and see what's more common across other Design Systems.